implementations/rust/provekit-cli/src/main.rs

```
@@ -162,7 +162,7 @@ pub struct ProveArgs {
     /// verifier is bypassed; instead the kit's lifter is spawned via JSON-RPC and
     /// each verifier in `provekit-self-contracts::lift_plugin_protocol` is run
     /// against the captured RPC messages.
-    /// Known kits: rust, go, cpp, ts, csharp, swift, java, python, ruby, zig, c.
+    /// Known kits: rust, go, cpp, ts, csharp, clr-bytecode, swift, java, python, ruby, zig, c.
```

Clippy (1.95.0)

[warning] the loop variable `j` is only used to index `lines`

(warning)